v-next/hardhat/src/internal/cli/init/init.ts

```diff
@@ -120,14 +120,14 @@ export async function initHardhat(options?: InitHardhatOptions): Promise<void> {
 // generated based on the "DOS Rebel" font
 function printAsciiLogo() {
   const logoLines = `
- █████  █████                          ███   ███                   ███       ████████
-░░███  ░░███                          ░███  ░███                  ░███      ███░░░░███
- ░███   ░███   ██████   ████████   ███████  ░███████    ██████   ███████   ░░░    ░███
- ░██████████  ░░░░░███ ░░███░░███ ███░░███  ░███░░███  ░░░░░███ ░░░███░       ██████░
- ░███░░░░███   ███████  ░███ ░░░ ░███ ░███  ░███ ░███   ███████   ░███       ░░░░░░███
- ░███   ░███  ███░░███  ░███     ░███ ░███  ░███ ░███  ███░░███   ░███ ███  ███   ░███
- █████  █████░░████████ █████    ░░████████ ████ █████░░████████  ░░█████  ░░████████
-░░░░░   ░░░░░  ░░░░░░░░ ░░░░░      ░░░░░░░░ ░░░░ ░░░░░  ░░░░░░░░    ░░░░░    ░░░░░░░░
+ █████  █████                         ███  ███                  ███      ██████
+░░███  ░░███                         ░███ ░███                 ░███     ███░░███
+ ░███   ░███   ██████  ████████   ███████ ░███████    ██████  ███████  ░░░  ░███
+ ░██████████  ░░░░░███░░███░░███ ███░░███ ░███░░███  ░░░░░███░░░███░      ████░
+ ░███░░░░███   ███████ ░███ ░░░ ░███ ░███ ░███ ░███   ███████  ░███      ░░░░███
+ ░███   ░███  ███░░███ ░███     ░███ ░███ ░███ ░███  ███░░███  ░███ ███ ███ ░███
+ █████  █████░░███████ █████    ░░███████ ████ █████░░███████  ░░█████ ░░██████
+░░░░░  ░░░░░  ░░░░░░░ ░░░░░      ░░░░░░░ ░░░░ ░░░░░  ░░░░░░░    ░░░░░   ░░░░░░
```

Comment:
The `3` is more narrow here. The `a`s and the `d` don't extend 1 cell to the right at the bottom anymore. The spacing between `t` and `3` is reduced.